sympy/concrete/summations.py

```diff
@@ -269,9 +269,54 @@ def _eval_difference_delta(self, n, step):
 
         return Sum(f, (k, upper + 1, new_upper)).doit()
 
-    def _eval_simplify(self, ratio, measure):
-        from sympy.simplify.simplify import sum_simplify
-        return sum_simplify(self)
+    def _eval_expand_mul(self, **hints):
```

Comment:
I agree.  Actually `sum_expand` itself should be called something else. In SymPy terminology, `sum_expand` is not expand, but factor.